datastore/importers/nmdb-import-dpkg/README.md

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION
+===========
+
+Parse and import the output of the `dpkg -l` command on modern Linux systems.
+
+TROUBLESHOOT
+============
+
+Parser is not designed to parse Unicode characters. If a package contains unicode characters in it's description, you will have to convert your `dpkg -l` output to ascii using a tool like `uni2ascii` which can be installed with `apt`.
```

Comment:
```suggestion
The tool will not handle Unicode characters.
If the output contains a Unicode character anywhere, the character will need to be converted to the ASCII equivalent, replace, or otherwise removed prior to processing.
This could be accomplished manually, by using a tool (e.g., `uni2ascii`), or similar.
```